### DISCUSSION TITLE
No longer able to authenticate -  "Jwt issuer is not configured" when testing API calls in reference

### DISCUSSION THREAD

**Discussion by pwlopez** [airbytehq/airbyte#44487] [Platform API Questions]
I was testing API calls via the reference and all of the sudden I can no longer authenticate and keep recieveing a code 401, "Jwt issuer is not configured".

**Comment by i0exception** [comment]
Same here!

**Comment by nataliekwong** [comment]
Hi there, are you using Cloud or OSS (Community or Enterprise)?

If Cloud or Enterprise, could you make sure to create a Support ticket if you have not already?

**Comment by assaadhjb** [comment]
Hi, Cloud for me, and Im running into the same issue...

**Comment by nataliekwong** [comment]
Hi @assaadhjb - we've investigated and discovered a fix, and this should be resolved now.  Can you try again now?

If you are still having issues, please submit a Support ticket to our Support team to further triage.

**Comment by assaadhjb** [comment]
Still an issue 😕 

After getting succesfully my-token from the request bellow:
```
curl --request POST \
     --url https://api.airbyte.com/v1/applications/token \
     --header 'accept: application/json' \
     --header 'content-type: application/json' \
     --data '
{
  "client_id": "my_client_id",
  "client_secret": "my_client_Secret"
}'
```
I run

```
curl --request GET \
--url https://api.airbyte.com/v1/connections \
--header 'accept: application/json' \
--header 'content-type: application/json' \
--header 'Authorization: Basic my-token'
```

This was working, until yesterday where I start receiving:

```{
    "code": 401,
    "message": "Jwt is missing"
}
```

**Reply by nataliekwong** [reply]
Can you try this with a newly created application?

**Reply by timroes** [reply]
@assaadhjb Your `Authorization` header is wrong. It shouldn't be `Basic <token>`, but `Bearer <token>`. You're trying to initiate a HTTP Basic Auth (which we don't support).

Please let me know if you still seeing this issue after putting a `Authorization: Bearer <token>` header on your request.

**Reply by assaadhjb** [reply]
@nataliekwong @timroes thanks both. Now it's working! 
Sorry, Idk if it relates to Cloud users as well, but I think the Basic auth [in the doc](https://reference.airbyte.com/reference/authentication#airbyte-oss-users) for OSS users is a bit confusing

**Reply by timroes** [reply]
@assaadhjb Thank you for the feedback. This document is indeed outdated and state the old state before we build authentication in Airbyte onto the new system. I'll make sure we're adopting this to reflect the correct current solution.

**Reply by joeykmh** [reply]
The [docs](https://reference.airbyte.com/reference/authentication) have been updated, thanks for the feedback @assaadhjb!

[a comment by an automated account is not shown]
